Products/CMFPlone/resources/eventhandlers.py

```diff
@@ -0,0 +1,16 @@
+from Products.CMFPlone.resources.browser.resource import update_resource_registry_mtime
+from Products.GenericSetup.interfaces import IProfileImportedEvent
+from zope.component import adapter
+
+
+@adapter(IProfileImportedEvent)
+def check_registry_update(event):
+    """Check if a profile import may have updated the configuration registry.
```

Comment:
We don't know if there was really a change in the resource registry.  The only thing we can check here, is if the `plone.app.registry` step was run, either explicitly or because a full import was done.

I almost wanted to change this comment anyway, but I see the rest of the docstring explains the current use case, and gives room for future use cases that are not concerned with the resource registries.  Maybe in the future we want to clear the RAM cache when anything in the configuration registry changes, or we want to update the modification date of the site root.

Reply:
Technically, the resource registry is just a setting in the configuration registry.

Reply:
Nice idea, but with a volatile cache this wont work in setups with more than one zooe client running.

Reply:
You're right. Then we've to live with a small write on read when theme resources changes. Part of the cache key can be a value in the registry, so updating this value here, the other threads cache will be updated on the next view... I don't think there can be a real solution.